modules/setting-up-posix-shared-memory.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-ephemeral-storage.adoc
+
+[id="setting-up-posix-shared-memory_{context}"]
+= Setting up POSIX Shared Memory
+
+
+The following procedure shows how to set up POSIX shared memory between two containers.
+
+.Procedure
+
+1. Edit the pod YAML file to specify the tmps volume for the containers, as shown in the `shared-memory.yaml` file.
++
+====
+.shared-memory.yaml
+[source,yaml]
+----
+---
+apiVersion: v1
+id: hello-openshift
+kind: Pod
+metadata:
+  name: hello-openshift
+  labels:
+    name: hello-openshift
+spec:
+  volumes:                          <1>
+    - name: dshm
+      emptyDir:
+        medium: Memory
+  containers:
+    - image: kubernetes/pause
+      name: hello-container1
+      ports:
+        - containerPort: 8080
+          hostPort: 6061
+      volumeMounts:                 <2>
+        - mountPath: /dev/shm
+          name: dshm
+    - image: kubernetes/pause
+      name: hello-container2
+      ports:
+        - containerPort: 8081
+          hostPort: 6062
+      volumeMounts:                 <3>
+        - mountPath: /dev/shm
+          name: dshm
+----
+<1> specifies the tmpfs volume `dshm`.
+<2> enables POSIX shared memory for `hello-container1` via `dshm`.
+<3> enables POSIX shared memory for `hello-container2` via `dshm`.
+====
+
+2. Create the pod using the pod YAML file:
```

Comment:
Are we editing an existing pod or creating a new one? Step 1 says to _Edit_ the pod spec, which suggests the pod already exists. In which case, `oc create` won't work, I think.

Reply:
I have changed it to creating a pod, although I am not sure how it should be done. Reached out to the QE; awaiting response.

Reply:
@chinmayi-chandrasekar I missed this before. We should not use `via` in docs.  IBM Style Guide says to use _to refer to routing data_. 
Should htis be _using_ instead?
`"Enables POSIX shared memory for `hello-containerX` using ephemeral storage volume`dshm`
`Or
`"Enables POSIX shared memory for `hello-containerX` using`dshm``

Reply:
Addressed.